src/kixi/heimdall/handler.clj

```diff
@@ -54,6 +61,22 @@
       {:status 201 :body res}
       {:status 401 :body res})))
 
+(defn escape-html
+  "Change special characters into HTML character entities."
+  [text]
+  (.. #^String (str text)
+      (replace "&" "&amp;")
+      (replace "<" "&lt;")
+      (replace ">" "&gt;")
+      (replace "\"" "&quot;")))
```

Comment:
Isn't there some middleware (or a library or something to steal from another library) that handles this escaping? I worry that this might be a bit brittle.